webui/Makefile

```diff
@@ -22,9 +22,16 @@ get-deps:
 	$(MAKE) -C elm $@
 	$(MAKE) -C tests $@
 
-build:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
+.PHONY: build
+build: build-react build-elm
+
+.PHONY: build-react
+build-react:
+	$(MAKE) -C react build
+
+.PHONY: build-elm
+build-elm:
+	$(MAKE) -C elm build
```

Comment:
question: what does this change do? Is the issue that it doesn't have a PHONY target? In which case, wouldn't it just need a `.PHONY: build` to sit above the `build:` target?

Reply:
If I'm getting the question right, it is to set elm build and react build as dependencies of the webuil build step rather than parts of it. when they are independent dependencies then `make -j` can run them concurrently